packages/radio-group/README.md

```diff
@@ -150,4 +150,12 @@ Indicates whether or not the radio can be clicked by a user
 
 Text that appears adjacent to the radio input; label text.
 
+### default
+
+**Type:** `boolean`
+
+**Default:** `false`
+
+If RadioGroup component is uncontrolled, this determines what Radio will be active on first render.
```

Comment:
Something like this would be more clear:
"If RadioGroup is uncontrolled, the `default` property makes this `Radio` checked on the initial render."

Reply:
👍